samples/suit/smp_transfer/sysbuild/nrf54h20dk_nrf54h20_memory_map.dtsi

```diff
@@ -37,6 +37,6 @@
 	};
 
 	cpurad_recovery_partition: partition@8e000 {
-		reg = < 0x8e000 DT_SIZE_K(92) >;
+		reg = < 0x8e000 DT_SIZE_K(100) >;
```

Comment:
Increasing the recovery radio image size by such an amount is not really acceptable from our side.

Is there really no way to provide a pseudo random number generator for this case so that it can be used by the entropy generator? This image is intended to be minimal and absolutely no BLE security is neeed.

Reply:
The RNG is needed for the entropy generator - is there any possibility for the entropy generator to use the pseudo random number generator?

Reply:
What about adding the following code to the `smp_transfer/sysbuild/recovery_hci_ipc.overlay` file which would replace the real RNG with a PRNG for this image:

```
/ {
        chosen {
                zephyr,entropy = &prng;
        };

/delete-node/ psa-rng;

        prng: prng {
                compatible = "nordic,entropy-prng";
                status = "okay";
        };

};
```

Reply:
Discussed on teams, the conclusion is that I will indeed update this sample to use the PRNG node and I will create a jira for this to be replaced later.